boxoffice/static/js/views/admin_discount_policy.js

```diff
@@ -0,0 +1,542 @@
+
+import {fetch, post, scrollToElement, getFormParameters, getCsrfToken, updateBrowserHistory, urlFor} from '../models/util.js';
+import {DiscountPolicyTemplate} from '../templates/admin_discount_policy.html.js';
+import {SideBarView} from './sidebar.js';
+
+export const DiscountPolicyView = {
+  render: function ({org_name, search, page, size}={}) {
+    let url;
+    if (search) {
+      url = urlFor('search', {
+        scope_ns: 'o',
+        scope_id: org_name,
+        resource: 'discount_policy',
+        root: true,
+        search: search,
+        page: page,
+        size: size
+
+      });
+    } else {
+      url = urlFor('index', {
+        scope_ns: 'o',
+        scope_id: org_name,
+        resource: 'discount_policy',
+        root: true,
+        page: page,
+        size: size
+      });
+    }
+
+    const DEFAULT = {
+      showForm: true,
+      hideForm: false,
+      showLoader: true,
+      hideLoader: false,
+      priceBasedDiscount: 1,
+      couponBasedDiscount: 1,
+      usageCount: 1,
+      empty: ""
+    };
+
+    fetch({
+      url: url
+    }).done(({org_name, title, discount_policies, currency_symbol, total_pages, paginated, current_page}) => {
+      // Initial render
+      let discountPolicyComponent = new Ractive({
+        el: '#main-content-area',
+        template: DiscountPolicyTemplate,
+        data:  {
+          org: org_name,
+          title: title,
+          discountPolicies: discount_policies,
+          currency_symbol: currency_symbol,
+          paginated: paginated,
+          totalPages: total_pages,
+          currentPage: current_page,
+          size: 20,
+          items: '',
```

Comment:
Looks like `items` is used as an array, further down? If that's the case, it's better to initialized with `[]`.